### ISSUE TITLE
View devtools::document() intermediate files

### ISSUE THREAD

**Issue by mccroweyclinton-EPA** [r-lib/devtools#2200]
tl;dr : I would like the option to view the roxygen2 intermediate file created by calling build_manual() if there is an error.

I am writing a manual for a R package that I am creating using roxygen2. For some reason I am receiving an error message

 ```
Writing <package>.Rd
Converting Rd files to LaTeX .........
Warning in system(paste(shQuote(texi2dvi), if (quiet) "--quiet" else "",  :
  running command '"C:\PROGRA~1\MIKTEX~1.9\miktex\bin\x64\texify.exe"  --pdf "Rd2.tex" --max-iterations=20 -I "C:/PROGRA~1/R/R-36~1.2/share/texmf/tex/latex" -I "C:/PROGRA~1/R/R-36~1.2/share/texmf/bibtex/bst"' had status 1
Error : running 'texi2dvi' on 'Rd2.tex' failed

LaTeX errors:
! You can't use `macro parameter character #' in horizontal mode.
l.401 #
       # Not run:
? 
! Emergency stop.
! Emergency stop.
l.401 #
       # Not run:
End of file on the terminal!

!  ==> Fatal error occurred, no output PDF file produced!

Warning in system(paste(shQuote(texi2dvi), if (quiet) "--quiet" else "",  :
  running command '"C:\PROGRA~1\MIKTEX~1.9\miktex\bin\x64\texify.exe"  --pdf "Rd2.tex" --max-iterations=20 -I "C:/PROGRA~1/R/R-36~1.2/share/texmf/tex/latex" -I "C:/PROGRA~1/R/R-36~1.2/share/texmf/bibtex/bst"' had status 1
Error : running 'texi2dvi' on 'Rd2.tex' failed

LaTeX errors:
! You can't use `macro parameter character #' in horizontal mode.
l.401 #
       # Not run:
? 
! Emergency stop.
! Emergency stop.
l.401 #
       # Not run:
End of file on the terminal!

!  ==> Fatal error occurred, no output PDF file produced!

Error in running tools::texi2pdf()
 Error: Failed to build manual 
```

Each individual Rd file compiles without an issue. But for some reason LaTex fails when generating the full package manual. So I have no idea where the issue is occurring, my only hint is in the output :

```
l.401 #
           # Not run:
? 
```

This tells me that the error is on line 401. After looking at the source code of the devtools it appears that the build_manual function just concatenates all of the package's Rd files into one Rd file, converts that into a latex output file named Rd2.tex, then uses LaTeX to convert that into a pdf file. Since I do understand LaTex and yet I don't know where exactly in all of my many roxygen2 comments this issue is coming from it would make debugging this issue 10000x more easier if I could view the package document's intermediate (the Rd2.tex file) and manually view  that entire file through a LaTeX editor to determine what is causing this issue.

Telling me that the issue is on line 401 without allowing me to see the actual line is not helpful at all. Is there a way to view the intermediate Rd file?

P.S. I believe the issue has something to do with my use of the pound/hashtag sign in one of my @example but I have many functions in my code, trying to pinpoint which line in my code could be causing this would take forever without having some sort of starting point.

**Comment by jimhester** [comment]
You can run `R CMD Rd2pdf --no-clean` from a command line terminal to not remove temporary files.

**Comment by mccroweyclinton-EPA** [comment]
I am confused
How would that generate the Rd2.tex file?

```
R CMD Rd2pdf --no-clean
no inputs
```

**Comment by jimhester** [comment]
you have to give the command a package directory, if you are in your packages root then `R CMD Rd2pdf --no-clean .`
